app/presenters/return-requirements/check/returns-requirements.presenter.js

```diff
@@ -72,12 +84,12 @@ function _mapRequirement (requirement, index, purposes, points) {
     index,
     points: _mapPoints(requirement.points, points),
     purposes: _mapPurposes(requirement.purposes, purposes),
+    returnsCycle: _returnsCycle(requirement.returnsCycle),
```

Comment:
The function currently caters for the cycle not being recognised. This is a case of being 99.99999999999999999999% sure there will only ever being 2, so if it's not one it must be the other.

On that basis I think we can reduce this down to a one-liner (which is also how we've dealt with this elsewhere! 😬 )

```suggestion
    returnsCycle: requirement.returnsCycle === 'summer' ? 'Summer' : 'Winter and all year',
```

Reply:
Cool. I was very tempted to just do this  !